apps/fares/lib/format.ex

```diff
@@ -96,7 +99,8 @@ defmodule Fares.Format do
   def name(:premium_ride), do: "Premium Ride"
   def name(:invalid), do: "Invalid Fare"
 
-  @spec full_name(Fare.t()) :: String.t() | iolist
+  @spec full_name(Fare.t() | nil) :: String.t() | iolist
+  def full_name(nil), do: "Free"
```

Comment:
Why is this Free?

Reply:
Oh yeah sorry, I should have mentioned that. This is for a shuttle bus leg which doesn't have a fare because it is free. (And crashed before I added this case for.) There's a larger ticket which this is a first little step towards as well: https://app.asana.com/0/1134952491130998/1178333054166489